openedx/features/course_experience/static/course_experience/js/spec/CourseOutline_spec.js

```diff
@@ -0,0 +1,113 @@
+/* globals Logger, loadFixtures */
```

Comment:
Ditto loadFixtures, although it's less important because its only called in tests.